src/AutoUI/models/helpers.ts

```diff
@@ -204,15 +249,54 @@ export const generateSchemaFromRefScheme = (
 	};
 };
 
+export const getRefSchema = (schema: JSONSchema, refSchemePrefix: string) => {
+	const refScheme = parseDescriptionProperty(schema, 'x-ref-scheme') as
+		| string
+		| null;
```

Comment:
This case sounds very wrong, especially since `string` is not a return type of `parseDescriptionProperty`

Why do we do this?